docs/how-to/work-with-pinning-services.md

```diff
@@ -113,19 +166,25 @@ $ ipfs pin remote --help
 
 ## Create your own pinning service
 
-Obviously you aren't limited to a static list of pre-approved services. Any remote pinning service compatible with the [IPFS Pinning Service API](https://ipfs.github.io/pinning-services-api-spec) can be added as a custom pinning service — which also means that you can create your own! This might be useful in circumstances like:
+If you're interested in creating your own pinning service for personal or shared use, you have two options:
+
+1. [Generate a client and server from the OpenAPI spec](https://github.com/ipfs/pinning-services-api-spec#code-generation)
+
+1. [Reuse an existing solution](https://github.com/ipfs/pinning-services-api-spec#adoption) to reduce development time. 
+
+Learn more about the Pinning Service Spec](https://github.com/ipfs/pinning-services-api-spec).
 
-- Designating one of your own IPFS nodes to be a _personal pinning service_ as a preferred location for permanent storage.
-- Running a private pinning service for your friends or company.
-- Starting your own commercial pinning service.
 
-As noted above, your service must use the [IPFS Pinning Service API](https://ipfs.github.io/pinning-services-api-spec) in order to be interoperable with client behind `ipfs pin remote` commands.
+::: callout
 
+**Make your pinning service available to the community**
 
-::: tip
-If you're interested in creating your own pinning service for your own personal or shared use, you can [generate client and server from the OpenAPI spec](https://github.com/ipfs/pinning-services-api-spec#code-generation), reducing the development time, or [reuse an existing solution](https://github.com/ipfs/pinning-services-api-spec#adoption)
+If you'd like to make your custom pinning service available to all IPFS users, we welcome your submissions. Once you're ready to open the doors to the public, create a pull request against the [IPFS Web UI GitHub repository](https://github.com/ipfs-shipyard/ipfs-webui) in order to add it to the default list of pinning services that are displayed in the Desktop/Web UI Settings screen, and one of the core maintainers will be in touch.
```

Comment:
nit: people  should do due dilligence first, before they open a PR, let's mention compliance test suite here :pray: 

```suggestion
If you'd like to make your custom pinning service available to all IPFS users, we welcome your submissions. 

First, make sure your service works correctly by running and passing [pinning-service-compliance](https://github.com/ipfs-shipyard/pinning-service-compliance) test suite.

Once you're ready to open the doors to the public, create a pull request against the [IPFS Web UI GitHub repository](https://github.com/ipfs-shipyard/ipfs-webui) in order to add it to the default list of pinning services that are displayed in the IPFS Desktop/Web UI Settings screen, and one of the core maintainers will be in touch.
```